### ISSUE TITLE
Docker compose requires docker login

### ISSUE THREAD

**Issue by sxhysj** [castlamp/zenbership#169]
I am planing to use Zenbership with Docker Compose. But it requires docker login as showing in the attached quote. Could you please let me know how to download the corresponding images? Many thanks. @Kenji-K 

output from running command: sudo docker-compose up -d
```
Pulling zenbership (castlamp/zenbership:latest)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling zenbership (castlamp/zenbership:latest)...
ERROR: pull access denied for castlamp/zenbership, repository does not exist or may require 'docker login'
```

**Comment by Kenji-K** [comment]
I am not aware of a Zenbership docker image having being published to the docker repo. What you must do is build the image yourself by using the supplied Dockerfile using the corresponding name (i.e. castlamp/zenbership).

Unfortunately I haven't been able to retest this since I submitted it, so any breaking changes with Docker might force you to tinker with it a bit. 

Also, at the time I submitted this pull request it had a pretty shiny "Works on my Machine™" seal of approval, but it seemed to have [some issues still](https://github.com/castlamp/zenbership/pull/71). Not sure if it got solved internally or not by the castlamp team (a part of which I am not really), but it eventually got approved without me having a chance to review properly.

Use at your own peril!
